arns/arns-token-1.md

```diff
@@ -72,6 +79,8 @@ The **ARNS-TOKEN-1** specification leverages the general AO Token and Subledger
 -- ARNS-TOKEN-1 Objects
```

Comment:
Recommend using "Mappings" or "Tables" - lua doesnt have "Objects" technically.

Reply:
clarified things in certain areas as tables or encoded JSON.  Left some of the main headings as "Objects" as updating this requires updating all of the specs to remain consistent.